seaice/pipeline/helpers.py

```diff
@@ -0,0 +1,38 @@
+"""Helper functions for running pipelines."""
+
+
+def jprint(msg):
```

Comment:
Is this a normal pattern in Jupyter notebooks?  Is the intent here to (more or less) write an ongoing log file?

Reply:
Logging is the intent, and also seeing the print statements in the notebook and in the terminal so that when the entire notebook is ran via an `anaconda project ...` command we have information coming back. Let's add logging with an actual logger in the future because we do want the stdout and stderr streams captured.

Reply:
[Ticketed](https://github.com/ua-snap/pipeline-template/issues/1) in the pipeline-template repo